arc-core/src/io/anuke/arc/files/FileHandle.java

```diff
@@ -137,7 +138,7 @@ public String absolutePath(){
 
     /** @return the name of the file, without any parent paths. */
     public String name(){
-        return file.getName();
+        return file.getName().isEmpty() ? file.getPath() : file.getName();
```

Comment:
The Windows roots don't have a `getName()`, so we fall back on `getPath()`, which I think is a reasonable change in any case.